keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -174,6 +174,17 @@ should be preventable when this KEP is in place.
     tenants.  Each user belongs to exactly one of the
     tenant-identifying groups.
 
+This KEP introduces a new functionality that is supposed to address the
+problems above. The goals/requirements for the solution (in the priority
+order) are the following:
+1. overload protection
+1. fairness
+1. throughput
```

Comment:
I do not agree that we have a strict ordering here.  I do not thing it is appropriate to stray far from being work-conserving in order to gain a tiny increment in fairness.